lcls-twincat-motion/Library/Version/Global_Version.TcGVL

```diff
@@ -1,7 +1,9 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
-<TcPlcObject Version="1.1.0.1" ProductVersion="3.1.4020.14">
+<TcPlcObject Version="1.1.0.1" ProductVersion="3.1.4022.18">
   <GVL Name="Global_Version" Id="{e148b39d-0d10-4eb7-9d77-80ffb59404c5}">
     <Declaration><![CDATA[{attribute 'TcGenerated'}
+{attribute 'no-analysis'}
+{attribute 'linkalways'}
```

Comment:
side question: do you know what these attributes are?

Reply:
– for excluding programming objects (e.g. POU, GVL, DUT) from the static analysis (coding rules, naming conventions, forbidden symbols)
– can only be used for whole programming objects

- The [linkalways] pragma causes the associated object to be selected by the compiler and is thus always contained in the compiler information. This means that the object is always compiled and loaded on the PLC. The option Always link in the object properties, category Advanced, has the same effect.

Reply:
Interesting, thanks for the explanation